tests/ListTest.php

```diff
@@ -191,7 +191,7 @@ function getKey() {
 
                     $a = [getKey() => 1];
                     takesList($a);',
-                'error_message' => 'MixedArgumentTypeCoercion',
+                'error_message' => 'InvalidArgument',
```

Comment:
This *could* be wrong — some form of argument type coercion, since it's coercing a list with a typed array with a key that could be 0. But OTOH if you wanted a simple list you'd remove the `getKey()` call altogether. So I'll allow it :)